.github/workflows/nolints.yml

```diff
@@ -42,12 +42,7 @@ jobs:
         with:
           auto-config: false
           use-github-cache: false
-          use-mathlib-cache: false
-
-      - name: get cache
-        run: |
-          lake exe cache clean
-          lake exe cache get
+          use-mathlib-cache: true
```

Comment:
I guess it wasn't doing anything anyways since this is run on a github hosted runner, but `lean-action` doesn't run `lake exe cache clean` before `lake exe cache get`.

(I suspect it was just left over when this was extracted from the main build action. The `cleanup` step also is probably unnecessary.)